ui/src/components/Footer.tsx

```diff
@@ -13,7 +13,7 @@ export default function Footer() {
 
   const ref = () => {
     if (info?.isRelease && info?.version) {
-      return `v${info.version}`;
+      return info.version;
```

Comment:
`info.version` should come (indirectly) from `metadata.Server.GetInfo`, so should now have the "v" prefix already, passed down as-is from `main`.